plugins/org.eclipse.sirius.diagram.ui/src-gen/org/eclipse/sirius/diagram/ui/provider/DiagramUIPlugin.java

```diff
@@ -641,6 +641,10 @@ public boolean removeCacheImage(ImageDescriptor desc) {
         }
 
         private boolean removeCacheImage(ImageWithDimensionDescriptor realDesc) {
+            Image image = getImage(realDesc);
```

Comment:
This line will create the image if it doesn't exist. Unuseful for a "remove" method.

Proposition:
```
            Image image = descriptorsToImages.remove(realDesc);
            if (image == null) {
                return false;
            } else {
                image.dispose();
                return true;
            }
```

Reply:
done